images/adapter/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:stable
```

Comment:
Can you instead use a smaller base image, such as `quay.io/gravitational/debian-tall:stretch`? I don't think container that runs a single binary needs full Debian.

Same for other Docker images (if you have any more, I couldn't see any though).

Reply:
will do and yes no other images here

Reply:
applied